dist/rpm/os-autoinst-openvswitch-test.spec

```diff
@@ -5,7 +5,7 @@ Version:        4.6
 Release:        0
 Summary:        test package for %{short_name}
 License:        GPL-2.0-or-later
-BuildRequires:  %{short_name}
+BuildRequires:  %{short_name} == %{version}
```

Comment:
For some reason this is commented out in 
https://github.com/os-autoinst/os-autoinst/blob/master/dist/rpm/os-autoinst-test.spec#L8
```
#BuildRequires:  %{short_name} == %{version}
BuildRequires:  %{short_name}
```
@okurz since you added this - any idea why?